src/screens/Users/InviteBanner.js

```diff
@@ -50,9 +47,7 @@ const HeaderWrapper = styled.View`
   justify-content: center;
   position: relative;
   height: 136px;
-  background-color: ${({ theme }) => getThemeType(theme) === DARK_THEME
-    ? theme.colors.tertiary
-    : theme.colors.basic070};
+  background-color: ${({ theme }) => theme.colors.basic060};
```

Comment:
Minor thing, but in light mode the background is darker than intended, like it was a part of the shadow instead of the card.

![image](https://user-images.githubusercontent.com/25530572/99804412-407e0e80-2b3b-11eb-9b97-1f52175f84da.png)

Reply:
Oh, yeah. Conditional color is needed here. Will adjust, thanks!